### PR TITLE
Prevents false positives when long method names are provided

### PR DESCRIPTION
**Pull request by oalmali**
This PR contains `WordCheck` object which basically checks if a text contains English words or not. It does this by reading `words` file which exist on all Unix and Unix-like operating systems ([see for more](https://en.wikipedia.org/wiki/Words_(Unix))). Then it compares the given text with all of the words in `words` file.

What we achieve by adding `WordCheck` is that we can avoid some false positives when really long method names are provided as file content. Since those method names or texts has high entropy because of their lengths, file detector was marking them as suspicious. Now we are adding one more step to make sure even if those texts has large entropy are not going to be marked as suspicious anymore.

### CODE REVIEW

**Comment by kelseymok** [comment]
:shipit:

File: detector/word_check.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+		word, err := reader.ReadString('\n')
+		word = strings.Trim(word, "\n")
+
+		if word != "" && len(word) > 2 && strings.Contains(text, word) {
+			text = strings.Replace(text, word, "", 1) //already matched
+			wordCount++
+		}
+
+		if err != nil { //EOF
+			fmt.Println("wordCount, ", wordCount)
```
Comment (jacksingleton) [inline detector/word_check.go]: should this be removed?

Comment (vishwas-tw) [reply detector/word_check.go]: Now fixed in another PR.

**Comment by vishwas-tw** [comment]
Please see #48 .

**Pull request closed without merging**